manifests/d/Decred/Decrediton/1.7.7/Decred.Decrediton.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Decred.Decrediton
+PackageVersion: 1.7.7
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/decred/decred-binaries/releases/download/v1.7.7/decrediton-v1.7.7.exe
+  InstallerSha256: 7727AEC7074652C0DF919DB68498B0F8D21D1598ABFB609643E7010526C9AE01

```

```suggestion
  InstallerSha256: 7727AEC7074652C0DF919DB68498B0F8D21D1598ABFB609643E7010526C9AE01
  ProductCode: '294af90e-a14b-58c1-adf0-2e358411a479'
```
Regardless of whether the string isn't used officially, the current installer does write `Decred Team` to Registry's Publisher value. WinGet relies on metadata from the registry to match upgrades and uses heuristics like DisplayName, Publisher, ProductCode to get a match for a package. If you don't want to have the registry's value in the manifest, I would suggest adding ProductCode to just be a little extra careful so that WinGet matching doesn't have a problem finding the package